package.json

```diff
@@ -31,7 +31,8 @@
   "homepage": "https://graffy.js.org",
   "dependencies": {
     "@babel/runtime": "^7.10.4",
-    "@babel/runtime-corejs3": "^7.10.4"
+    "@babel/runtime-corejs3": "^7.10.4",
+    "graphiql": "^1.4.2"
```

Comment:
:+1: but let's move it to devDependencies.

This is a monorepo from which multiple packages are published, so I have a script that generates the correct package.json for each package at the time of publishing. In this global package.json I use the `dependencies` field for dependencies of all the @graffy packages; If something is used only in certain packages, put it in devDependencies.